src/arch/x86_64/CodeGen.zig

```diff
@@ -382,8 +382,32 @@ fn gen(self: *Self) InnerError!void {
             .data = undefined,
         });
 
+        // push the callee_preserved_regs that were used
+        _ = try self.addInst(.{
+            .tag = .push_regs_from_callee_preserved_regs,
+            .ops = undefined,
+            .data = undefined, // to be backpatched
+        });
+        const backpatch_push_callee_preserved_regs_i = self.mir_instructions.len;
```

Comment:
```suggestion
        const backpatch_push_callee_preserved_regs_i = try self.addInst(.{
            .tag = .push_regs_from_callee_preserved_regs,
            .ops = undefined,
            .data = undefined, // to be backpatched
        });
```

Then you can use the index (without `- 1`) in the following code.